_inc/client/at-a-glance/activity.jsx

```diff
@@ -4,34 +4,37 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import DashItem from 'components/dash-item';
-import Card from 'components/card';
 import { translate as __ } from 'i18n-calypso';
-// import { get, includes } from 'lodash';
+import { isEmpty, get } from 'lodash';
 import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
+import DashItem from 'components/dash-item';
```

Comment:
Do you think you could take the opportunity to sort all these in alphabetical order, so it's easier to skim through in the future?

Reply:
Done.